examples/function-appaction/build-functions.js

```diff
@@ -61,13 +63,15 @@ const main = async (watch = false) => {
     const config = {
       entryPoints: getEntryPoints(),
       bundle: true,
-      minify: true,
-      platform: 'node',
       outdir: 'build',
-      logLevel: 'info',
       format: 'esm',
-      target: 'es6',
-      external: ['node:*'],
```

Comment:
I believe we need to retain `'node:*'` imports as external such that it builds without those dependencies and expect them to be provided by the runtime.

Reply:
I dont think it's needed anymore now that we include prebuilt node modules. I just confirmed by including it in the configuration and the file checksum is the same